packages/base/any/kernels/3.16-lts/patches/driver-igb-netberg-aurora.patch

```diff
@@ -0,0 +1,599 @@
+diff -Nu a/drivers/net/ethernet/intel/igb/bcm_phy.c b/drivers/net/ethernet/intel/igb/bcm_phy.c
```

Comment:
The function "bcm54616s_linkup" and "bcm54616_config_init" in "bcm_phy.c" is needed for Netberg platforms and we will remove unused and re-define definitions in this file.